### PR TITLE
DR | Update Appeals Forms with USWDS v3

### PR DESCRIPTION
**Pull request by rocio-desantiago**
## Summary

- _(Summarize the changes that have been made to the platform)_
  > Updating web components in shared folders used across the 3 appeal forms: Supplemental Claims, Higher Level Review, and Notice of Disagreement (Board Appeals) and the testing (prototype for NOD) to use USWDS v3 styling. This is the first step to migrating all forms to the latest styling.

  > Update structure of `VaBreadcrumbs`.
  > Add `uswds` property to following v3 web components: `va-alert`, `VaTextInput`, `va-button`, `VaCheckboxGroup`, `VaModal`, and `va-additional-info`.
  > remove `uswds` from `va-on-this-page` & `va-telephone`- previously thought this was an updated component, confirmed it's not.

- _(Which team do you work for, does your team own the maintenance of this component?)_
  > Benefits Decision Reviews

- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Related issue(s)

[#73827](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73827)

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

N/A - v3 styles & colors are different

## What areas of the site does it impact?

Supplemental Claim

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/appeals/download/containers/ViewAppeal.jsx [hunk 1]
```diff
@@ -168,6 +184,7 @@ const ViewAppeal = () => {
           class="screen-only"
           onClick={window.print}
           text="Print this page for your records"
+          uswds
```
Comment (rocio-desantiago) [inline src/applications/appeals/download/containers/ViewAppeal.jsx:187]: @Mottie this might be a lingering `uswds`. Does the subtitle div need it?

Comment (Mottie) [reply src/applications/appeals/download/containers/ViewAppeal.jsx:187]: This `uswds` is added to a `va-button`, so it's fine. Expand the code up.

**Review by Mottie — APPROVED** [review]
LGTM!

**Review by rsmithadhoc — APPROVED** [review]

File: src/applications/appeals/download/containers/ViewAppeal.jsx (hunk 1 above)
Comment (rocio-desantiago) [reply src/applications/appeals/download/containers/ViewAppeal.jsx:187]: Ope that's right. Was looking at the wrong beginning of code.

**Pull request merged**